### PR TITLE
Add node.js mock server

### LINKED ISSUE #34: Mock metrics server
In order to continue with the frontend development we are gonna have a mock node backend that serves sample data, which is gonna follow the models that later will be presented by the actual Freestyle Opscenter project. This will help us to fulfill frontend side objectives, for testing purposes, etc.

The reason to use node for this mock, is to be able to get it launched inside the same actual ecosystem. Also, besides that, we need to take into account that ideally this mocks would be serving proto encoded data.

### PR DESCRIPTION
**Pull request by calvellido**
* Add node.js code and directory
* Update README

This closes #34

### CODE REVIEW

File: mock-server/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "start": "node index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "server",
+    "websocket",
+    "websockets"
+  ],
+  "author": "Juan Valencia Calvellido",
+  "license": "MIT",
```
Comment (raulraja) [inline mock-server/package.json]: This should remain the same License as the project ASL 2

Comment (calvellido) [reply mock-server/package.json]: Addressed in f01481f

File: mock-server/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    console.log(msg);
+  });
+  ws.on('close', function () {
+    console.log('disconnection');
+  });
+  console.log('socket', req.testing);
+});
+
+setInterval(function () {
+  expressWs.getWss('/').clients.forEach(function (client) {
```
Comment (eperinan) [inline mock-server/index.js]: The websocket path is wrong. Here you can see the proper path `/metrics`. https://github.com/frees-io/freestyle-opscenter/blob/master/core/src/main/scala/runtime/endpoints.scala#L77

**Review by eperinan — APPROVED** [review]
LGTM, but please, address the comment that I put.

File: mock-server/index.js (hunk 2 above)
Comment (calvellido) [reply mock-server/index.js]: Addressed in 45e5213cd806a471501424b96d31662387895ae5

**Pull request merged**